### PR TITLE
Js menu

### PR DESCRIPTION
**Pull request by Jeanbulambo**
## Mobile Menu
 In this project done by Jean Bulambo and Daniel Galindo, it has been uploaded the following changes:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -27,6 +30,10 @@ ul {
   height: 12px;
 }
 
+.hambmobzz {
```
Comment (OmarRamoun) [inline style.css]: - [ ] [REQUIRED] Please, make sure that the hamburger menu doesn't show on desktop version as requested in project requriements :receipt:

> As you can see below on the top left the humbarger icon shows
image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const nav = document.querySelector('.mobile');
+const hamb = document.querySelector('.menu');
+
+hamb.addEventListener('click', () => {
+  hamb.classList.toggle('active');
+  nav.classList.toggle('active');
```
Comment (OmarRamoun) [inline script.js:7]: You have manged to add an event listener to the humberger icon on `click` :clap:. But unfortunately you forgot to add event listeners to each element in the nav list.

- [ ] [REQUIRED] Please, make sure to make the nav menu disappear when you click on one of its items as requested in project requirements.

> So far, when one of the items is clicked it navigates to the relative section but the menu doesn't disappear as it shows in the following picture :arrow_down:.
image

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @Jeanbulambo 👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — APPROVED** [review]
Hi @Jeanbulambo,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**